### ISSUE TITLE
Upgrade the version of bench-show in chart executable

### ISSUE THREAD

**Issue by harendra-kumar** [composewell/unicode-transforms#35]
I was thinking that we are using new `bench-show` package and in all benchmark results I wrote that it shows the % diff from the lower value, but it actually show percent diff from the baseline always, because of the older version.

need bench-show upgrade to 0.3.1

**Comment by harendra-kumar** [comment]
We may also have to upgrade bench-show to newer ghc 8.10.

**Comment by harendra-kumar** [comment]
`git blame` says I fixed it in July last year via commit a48b3b58:

```
a48b3b58 (Harendra Kumar  2020-07-28 23:40:53 +0530 203)       , bench-show >= 0.3 && < 0.4
```
